_sass/layout/_topbar.scss

```diff
@@ -6,6 +6,7 @@
 #topbar-wrapper {
   height: v.$topbar-height;
   background-color: var(--topbar-bg);
+  flex-shrink: 0;
```

Comment:
```suggestion
```

I think adding Bootstrap class to the HTML would be easier to maintain!

https://github.com/cotes2020/jekyll-theme-chirpy/blob/db40db6d3df7058a28b04f42ff6ee79954a5de1c/_includes/topbar.html#L3



```diff
- <header id="topbar-wrapper" aria-label="Top Bar">
+ <header id="topbar-wrapper" class="flex-shrink-0" aria-label="Top Bar">
```

Reply:
Replaced, thank you.